### PR TITLE
Feat/checkout

### PR DESCRIPTION
**Pull request by pedromtec**
#### What does this PR do? \*

it allows to use the component at checkout.

#### How to test it? \*
Go to checkout and buy for  `pilateslovers` account.
[workspace](https://pedrocheckout--extensions.myvtex.com/agencian1-n1insightpro/p)

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -15,11 +15,10 @@
...
     "vtex.formatted-price": "0.x",
     "vtex.styleguide": "9.x",
-    "vtex.css-handles": "0.x"
+    "vtex.css-handles": "0.x",
+    "vtex.order-manager": "0.x"
   },
-  "registries": [
-    "smartcheckout"
-  ],
+  "registries": ["smartcheckout"],
```
Comment (arturpimentel) [inline manifest.json:21]: This is unnecessary
```suggestion
```

File: react/AppStorePrice.tsx [hunk 2]
```diff
@@ -149,9 +152,10 @@ interface AppPriceQuery {
 }
 
 const AppStorePrice: FunctionComponent = () => {
-  const {
-    product: { productId },
-  } = useProduct()
+  const { product } = useProduct()
+  const { orderForm } = useOrderForm()
+
+  const productId = product ? product.productId : orderForm.items[0].productId
```
Comment (arturpimentel) [inline react/AppStorePrice.tsx:158]: Maybe we could isolate responsibilities by making two different components:
```jsx
function CheckoutPrice() {
  const {
    orderForm: {
      items: [{ productId }]
    }
  } = useOrderForm()

  return <AppStorePrice productId={productId} />
}

function ProductPrice() {
  const {
    product: { productId },
  } = useProduct()

  return <AppStorePrice productId={productId} />
}

function AppStorePrice({ productId }) {
  // old logic here
}
```
Then you could use a different block for each component and place them accordingly in our theme

**Review by arturpimentel — CHANGES_REQUESTED** [review]

File: react/AppStorePrice.tsx (hunk 2 above)
Comment (pedromtec) [reply react/AppStorePrice.tsx:158]: It makes sense, but we need to release a major version.

**Review by marlonguimaraes — APPROVED** [review]

**Comment by pedromtec** [comment]
> #### What does this PR do? *
> it allows to use the component at checkout.
> 
> #### How to test it? *
> Go to checkout and buy for `pilateslovers` account.
> [workspace](https://pedrocheckout--extensions.myvtex.com/agencian1-n1insightpro/p)

**Review by arturpimentel — APPROVED** [review]

**Pull request merged**